packages/kuma-gui/src/app/gateways/routes.ts

```diff
@@ -62,6 +62,16 @@ export const routes = () => {
               path: 'builtin',
               name: 'builtin-gateway-list-view',
               component: () => import('@/app/gateways/views/BuiltinGatewayListView.vue'),
+              children: [
+                {
+                  path: ':gateway',
+                  name: 'builtin-gateway-summary-view',
+                  props: () => ({
+                    routeName: 'builtin-gateway-summary-view',
+                  }),
```

Comment:
Oh sorry missed this, I think I added a PR that made it so you no longer need to add these here, they are added automatically. Unfortunately you still need the `defineProps<{routeName: string}>()` in the route component for the moment. (gonna fire off a single comment, and keep reviewing but I think this is the last one and I can approve)

Reply:
Ah, right 🙂